src/wallet/KintoWallet.sol

```diff
@@ -68,9 +69,10 @@ contract KintoWallet is Initializable, BaseAccount, TokenCallbackHandler, IKinto
 
     /* ============ Constructor & Initializers ============ */
 
-    constructor(IEntryPoint __entryPoint, IKintoID _kintoID) {
+    constructor(IEntryPoint __entryPoint, IKintoID _kintoID, IKintoAppRegistry _kintoApp) {
```

Comment:
```suggestion
    constructor(IEntryPoint __entryPoint, IKintoID _kintoID, IKintoAppRegistry _registry) {
```

Reply:
or `_appRegistry` to be consistent with the other variables like kintoID

Reply:
done